tools/memleak.py

```diff
@@ -223,12 +203,12 @@ def run_command_get_pid(command):
         sizes.delete(&pid);
 
         info.timestamp_ns = bpf_ktime_get_ns();
-        info.num_frames = grab_stack(ctx, &info) - 2;
```

Comment:
Yep, that was a workaround. I don't remember the exact details anymore :-) ...but I think memleak would out irrelevant values in the last two entries. If we can grab the full stack without any of these hacks, I'm all for it.